sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfigurationTest.java

```diff
@@ -48,7 +49,7 @@ void configureReader_PrometheusOnClasspath() {
 
     MetricReader reader =
         MetricExporterConfiguration.configureReader(
-            "prometheus", EMPTY, spiHelper, (a, b) -> a, closeables);
+            "prometheus", CONFIG_PROPERTIES, spiHelper, (a, b) -> a, closeables);
     cleanup.addCloseables(closeables);
```

Comment:
All these tests cleanup their exporters / readers. This change will help make this test more resilient against other tests starting prometheus on the default port and not cleaning up, but best case we identify / fix the test failing to clean up.